src/components/pages/InntektStep.tsx

```diff
@@ -115,22 +115,22 @@ const InntektStep = () => {
             ----
           </option>
           <option value={'20'} key="20">
-            20%
+            20&nbsp;%
           </option>
           <option value={'40'} key="40">
-            40%
+            40&nbsp;%
           </option>
           <option value={'50'} key="50">
-            50%
+            50&nbsp;%
           </option>
           <option value={'60'} key="60">
-            60%
+            60&nbsp;%
           </option>
           <option value={'80'} key="80">
             80%
```

Comment:
```suggestion
            80&nbsp;%
```
(Strengt tatt kan man bruke et vanlig mellomrom i `<option>` siden teksten aldri vil wrappe uansett, men begge deler funker.)